kernel/arch/dreamcast/include/dc/fs_vmu.h

```diff
@@ -51,6 +51,7 @@ int fs_vmu_shutdown(void);
 #define IOCTL_VMU_SET_HDR          0x564d5530 /* "VMU0" */
 #define IOCTL_VMU_GET_REALFSIZE    0x564d5531 /* "VMU1" */
 #define IOCTL_VMU_GET_HDR_STATE    0x564d5532 /* "VMU2" */
+#define IOCTL_VMU_GET_INTLDATGM    0x564d5533 /* "VMU3" */
```

Comment:
Nitpick, but I'm not fond of the macro name, `INTLDATGM` doesn't mean much to me. What about `INIT_DATA`?